src/Firehose.Web/Web.config

```diff
@@ -37,10 +36,6 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
```

Comment:
Why did you remove this?

Reply:
I think it was removed automatically by VS for Mac. I'll add it back and re-submit

Reply:
You always have the possibility of ignoring files in a commit 😄